cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/ExecutionSparkContext.java

```diff
@@ -267,15 +274,13 @@ public ExecutionSparkContext(ApplicationSpecification appSpec,
 
   @Override
   public synchronized <T extends Dataset> T getDataset(String name, Map<String, String> arguments) {
-    Map<String, String> datasetArgs = RuntimeArguments.extractScope(Scope.DATASET, name, getRuntimeArguments());
```

Comment:
Why this is removed? So we don't do argument extraction from program runtime arguments anymore?